src/main/kotlin/org/polystat/j2eo/treeMapper/TreeMappings.kt

```diff
@@ -4,7 +4,138 @@ package org.polystat.j2eo.treeMapper
 import lexer.Token
 import lexer.TokenCode
 import org.antlr.v4.runtime.tree.TerminalNode
-import org.polystat.j2eo.antlrParser.JavaParser.*
+// import JavaParser.*
+import JavaParser.CompilationUnitContext
```

Comment:
@includealex Are all those imports used in the code? There should not be unused imports.

Reply:
@mximp yes, all the imports added are used in the code